sonar-project.properties

```diff
@@ -1,12 +1,14 @@
+sonar.organization=camel-tooling
 sonar.projectKey=vscode-camelk
 sonar.projectName=A VS Code extension supporting Camel-K
+
+# relative paths to source directories. More details and properties are described
+# in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/ 
 sonar.sources=src
 sonar.exclusions=src/test/**,src/ui-test/**
-sonar.tests=src/test,src/ui-test
-sonar.typescript.lcov.reportPaths=coverage/lcov.info
-sonar.organization=camel-tooling
+sonar.tests=src/test
 
 sonar.links.homepage=https://github.com/camel-tooling/vscode-camelk
-sonar.links.ci=https://app.circleci.com/pipelines/github/camel-tooling/vscode-camelk
+#sonar.links.ci=
```

Comment:
Why commenting instead fo replacing with the new location?

Reply:
Fixed